bin/build-rust-docs.sh

```diff
@@ -32,6 +32,41 @@ rm -rf build/docs
 mkdir -p build/docs
 echo '<meta http-equiv=refresh content=0;url=book/index.html>' > build/docs/index.html
 
+# Add redirections for all pages moved on the restructuring done on Bug 1708204
+# Android Specific Information
```

Comment:
When are we planning on removing these? Are we even considering removing them? (just out of curiosity, no action required)

Reply:
I would say we should remove them when this is at least a whole year in the past. This is completely a arbitrary timeline though.

Definitely nowhere in the near future.